### PR TITLE
Switch VEDA staging hub from EFS to jupyter-home-nfs server

### PR DESCRIPTION
**Pull request by sunu**
Changes include:
- Switch storage provider of VEDA staging hub from EFS to the in-cluster NFS server provided by jupyter-home-nfs
- Upgrade jupyter-home-nfs to the latest version
  - the new version supports specifying the parent path of the home directories for enforcing storage quota. (earlier the quota enforcement path was hard-coded to be `/export/`). For the staging hub, we set it to `/export/staging`.
  - The new version adds 'nfs' to k8s resource names, no longer relying only on Helm release names for identification

### CODE REVIEW

**Comment by sgibson91** [comment]
@sunu I don't know if you want to make changes here or just merge it and make a new PR for figuring out the NFS filepath mount. You have merge rights so I'll leave it up to you :)

**Comment by sunu** [comment]
@sgibson91 I've added a few more changes here. Added a note to the top comment describing the changes. Could you take another quick look please?

**Review by sgibson91 — APPROVED** [review]

**Pull request merged**
